source/api/discovery/0.5/index.md

```diff
@@ -279,15 +290,15 @@ As the number of Activities is likely too many to usefully be represented in a s
 
 The W3C [Activity Streams][org-w3c-activitystreams] specification defines a "model for representing potential and completed activities", and is compatible with the [design patterns][annex-patterns] established for IIIF APIs. It is defined in terms of [JSON-LD][org-w3c-json-ld] and can be seamlessly integrated with the existing IIIF APIs. The model can be used to represent activities carried out by content publishers of creating, updating, and deleting (or otherwise de-publishing) IIIF resources.
 
-This section is a summary of the properties and types used by this specification, and defined by Activity Streams.  This is intended to ease implementation efforts by collecting the relevant information together.
+This section is a summary of the properties and types used by this specification, and defined by Activity Streams. This is intended to ease implementation efforts by collecting the relevant information together.
 
-Properties, beyond those described in this specification, that the consuming application does not have code to process _MUST_ be ignored.  Other properties defined by Activity Streams _MAY_ be used, such as `origin` or `instrument`, but there are no current use cases that would warrant their inclusion in this specification.
+Properties, beyond those described in this specification, that the consuming application does not have code to process _MUST_ be ignored. Other properties defined by Activity Streams _MAY_ be used, such as `origin` or `instrument`, but there are no current use cases that would warrant their inclusion in this specification.
 
 
 ### 3.1. Ordered Collection
 {: #orderedcollection}
 
-The top-most resource for managing the lists of Activities is an Ordered Collection, broken up into Ordered Collection Pages. This is the same pattern that the Web Annotation model uses for Annotation Collections and Annotation Pages. The Collection does not directly contain any of the Activities, instead it refers to the `first` and `last` pages of the list.
+The top-most resource for managing the lists of Activities is an Ordered Collection, broken up into Ordered Collection Pages. The Collection does not directly contain any of the Activities, instead it refers to the `first` and `last` pages of the list.
```

Comment:
Web Annotation isn't mentioned anywhere else. Seems like a wasted sentence.

Reply:
True enough! It was a precedent that was discussed, but isn't important to understand the spec.  :+1: